benchexec/tools/fdse.py

```diff
@@ -0,0 +1,72 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.template
+import benchexec.result as result
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for FDSE.
+    https://github.com/zbchen/FDSE
```

Comment:
This link doesn't exist?

Reply:
This link is currently private.

Reply:
Is there a public URL for your tool? Or will this be made public?

Please remove the "(now is private)" comment from the code, it doesn't make sense to have it there because you would only need to file another pull request once the page becomes online.